internal/templates/common/cloudrun/deploy.sh.tmpl

```diff
@@ -0,0 +1,44 @@
+# Cloud Run deployment script for {{ .ADL.Metadata.Name }}
+# This script uses gcloud CLI for direct deployment without Kubernetes/Knative dependencies
+
+#!/bin/bash
+set -e
+
+# Configuration
+SERVICE_NAME="{{ .ADL.Metadata.Name }}"
+IMAGE_NAME="gcr.io/PROJECT_ID/{{ .ADL.Metadata.Name }}:{{ .ADL.Metadata.Version | default "latest" }}"
+PORT="{{ .ADL.Spec.Server.Port | default 8080 }}"
+REGION="${REGION:-us-central1}"
+PROJECT_ID="${PROJECT_ID}"
+
+# Build and push the Docker image
+echo "Building and pushing Docker image..."
+if command -v docker &> /dev/null; then
+  echo "Using Docker to build and push image..."
+  docker build -t "$IMAGE_NAME" .
+  docker push "$IMAGE_NAME"
```

Comment:
```suggestion
  echo "Using Docker to build and push image..."
```
Why do we need to rebuild ? we already have by now the ghcr oci image pushed - please use it. I don't think we need docker here actually.

Also we should allow to user to specify from what container image registry they want to deploy the serverless service.